hooks/pre_gen_project.py

```diff
@@ -8,7 +8,7 @@
 logging.basicConfig(level=logging.DEBUG)
 logger = logging.getLogger('pre_gen_project')
 
-PLUGIN_REGEX = r'^(?!pytest)[_a-zA-Z][_a-zA-Z0-9]+$'
+PLUGIN_REGEX = r'^(?!pytest)[-_a-zA-Z][-_a-zA-Z0-9]+$'
```

Comment:
I feel we should also have a check for ``module_name`` to be pep8 compliant. :smile: